frontend/src/pages/home/AlternatingBackgroundColorSectionList.tsx

```diff
@@ -2,45 +2,107 @@ import React from 'react';
+  color: string;
+  background: string;
+  padding: string;
+}) => {
+  const theme = useTheme();
+
+  return (
+    <Grid
+      item
+      xs={12}
+      sx={{
+        display: 'flex',
+        justifyContent: 'center',
+        padding: padding,
+        [theme.breakpoints.down('md')]: {
+          p: padding,
+          px: { xs: '16px', md: padding },
+        },
+        color: color,
+        background: background,
+      }}
+    >
+      {child}
+    </Grid>
+  );
+};
+
 const AlternatingBackgroundColorSectionList = ({
   children,
-  secondaryBackground = '#1282B2',
   secondaryColor = 'white',
+  secondaryBackground = '#1282B2',
+  itemPadding = '48px',
+  firstItemPadding = '32px',
+  lastItemIsPrimary,
 }: {
   children: React.ReactNode[];
-  secondaryBackground?: string;
   secondaryColor?: string;
+  secondaryBackground?: string;
+  // Used to wrap every item.
+  itemPadding?: string;
+  // Allow the first item to have a different padding.
+  firstItemPadding?: string;
+  // If true the last item will use the primary color regardless of its
+  // position in the list.
+  lastItemIsPrimary?: boolean;
 }) => {
   const theme = useTheme();
 
+  // children.at(-1) didn't work here...
+  const lastItem = children.slice(children.length - 1);
+
+  const nextColor = (idx: number): string => {
+    return idx % 2 == 0 ? secondaryColor : theme.palette.text.primary;
+  };
+
+  const nextBackground = (idx: number): string => {
+    return idx % 2 == 0
+      ? secondaryBackground
+      : theme.palette.background.primary;
+  };
+
   return (
-    <div
-      style={{
-        width: '100%',
-      }}
-    >
-      <Grid container>
-        {children.map((child: React.ReactNode, i: number) => (
-          <Grid
-            key={i}
-            item
-            xs={12}
-            sx={{
-              display: 'flex',
-              justifyContent: 'center',
-              padding: '32px',
-              [theme.breakpoints.down('md')]: {
-                padding: '32px 16px 32px 16px',
-              },
-              background: i % 2 == 0 ? secondaryBackground : '#FAF8F3',
-              color: i % 2 == 0 ? secondaryColor : 'black',
-            }}
-          >
-            {child}
-          </Grid>
-        ))}
-      </Grid>
-    </div>
+    <Grid container width="100%">
+      {/* display all items except the last one */}
+      {children.slice(0, -1).map((child: React.ReactNode, i: number) => (
+        <SectionGridItem
+          key={i}
+          child={child}
+          padding={i == 0 ? firstItemPadding : itemPadding}
+          color={nextColor(i)}
+          background={nextBackground(i)}
+        />
+      ))}
+
+      {/* display the last item */}
+      <SectionGridItem
+        child={lastItem}
+        padding={itemPadding}
+        color={
+          lastItemIsPrimary
+            ? theme.palette.text.primary
+            : children.length % 2 != 0
+            ? secondaryColor
+            : theme.palette.text.primary
```

Comment:
Could this be moved to the `nextColor` function and then not require to make a special case in the map on line 76 ?